leftwm/doc/leftwm.1

```diff
@@ -0,0 +1,339 @@
+The list of available subcommands available for leftwm is listed below:
+.SH SUBCOMMANDS
+.IP "-h, --help"
+Prints help information for this subcommand. This flag can also be used with other subcommands to display their own help message.
+.IP "-v --version"
+Prints the version information.
+.IP "check"
+This command will run several actions to ensure leftwm is configured properly, this will report the current leftwm version and git commit, it will also check if configuration is loaded correctly, check for syntax errors in your
+.I config.toml
+file, check your environment and theme files for errors as well.
+.IP "command"
+This subcommand sends commands directly to leftwm. It can also be used to concat commands for a keybind in
+.I config.toml
+For a list of available commands use the '-l|list' flag.
+.IP "state"
+Prints the current state of leftwm (in JSON format). You can also use flags and liqud-like syntax for a more refined output of this command.
+.IP "theme"
+Manage leftwm themes (This is part of an external package found in: https://GitHub.com/leftwm/leftwm-theme).
+.B "NOTE:"
+This subcommand will check for basic / common mistakes made in the directory structure or permissions, however it will not check or report if any mistakes were made in any extra scripts, bar configurations, liquid templates and anything that isn't the up or down executables.
+.SH CONFIGURING LEFTWM
+.IP "Configuration file"
+The settings file of leftwm can be found at:
+.I $HOME/.config/leftwm/config.toml
+.IP
+With this file you can configure the modkey for leftwm, available keybinds, tag names, workspace layouts and more.
+.RS
+NOTE: This file is automatically generated when leftwm or leftwm-check is run for the first time.
+.SS Default keybinds
+.IP "Movement"
+ up, down, left or right refer to the arrow keys in your keyboard.
+.TS
+tab(@);
+l l.
+T{
+Keybinding
+T}@T{
+Description
+T}
+_
+T{
+Mod + (1-9)
+T}@T{
+Switch to a desktop/tag
+T}
+T{
+Mod + Shift + (1-9)
+T}@T{
+Move the focused window to desktop/tag
+T}
+T{
+Mod + W
+T}@T{
+Switch the desktops for each screen.
+Desktops [1][2] changes to [2][1]
+T}
+T{
+Mod + Shift + W
+T}@T{
+Move window to the other desktop
+T}
+T{
+Mod + (up or down)
+T}@T{
+Focus on the different windows in the current workspace
+T}
+T{
+Mod + Shift + (up or down)
+T}@T{
+Move the different windows in the current workspace
+T}
+T{
+Mod + Enter
+T}@T{
+Move selected window to the top of the stack in the current workspace
+T}
+T{
+Mod + Ctrl + (up or down)
+T}@T{
+Switch between different layouts
+T}
+T{
+Mod + Shift + (left or right)
+T}@T{
+Switch between different workspaces
+T}
+T{
+Mod + Shift + Enter
+T}@T{
+Open a terminal
+T}
+T{
+Mod + Ctrl + L
+T}@T{
+Lock the screen
+T}
+T{
+Mod + Shift + X
+T}@T{
+Exit LeftWM
+T}
+T{
+Mod + Shift + Q
+T}@T{
+Close the current window
+T}
+T{
+Mod + Shift + R
+T}@T{
+Reload LeftWM and its config
+T}
+T{
+Mod + p
+T}@T{
+Use dmenu to start application
+T}
+.TE
```

Comment:
This table markup scares me. Could you change that to something like the following?
Maybe use a simpler markup like that on the other tables too?

```groff
.TS
tab(;);
l l.
Keybinding;Description
_
Mod + (1-9);Switch to a desktop/tag
Mod + Shift + (1-9);Move the focused window to desktop/tag
Mod + W;Switch the desktops for each screen. Desktops [1][2] changes to [2][1]
Mod + Shift + W;Move window to the other desktop
Mod + (up or down);Focus on the different windows in the current workspace
Mod + Shift + (up or down);Move the different windows in the current workspace
Mod + Enter;Move selected window to the top of the stack in the current workspace
Mod + Ctrl + (up or down);Switch between different layouts
Mod + Shift + (left or right);Switch between different workspaces
Mod + Shift + Enter;Open a terminal
Mod + Ctrl + L;Lock the screen
Mod + Shift + X;Exit LeftWM
Mod + Shift + Q;Close the current window
Mod + Shift + R;Reload LeftWM and its config
Mod + p;Use dmenu to start application
.TE
```

Reply:
I'm no groff / troff expert, will that render the same on terminal? :o

Reply:
haha, me neither :)
Yeah it should, would probably be good if you could do a quick comparison to make sure.

Reply:
Here's a quick roundup what the markup means as I understand it, but correct me if I'm wrong because I learned that by doing, not by reading the spec :stuck_out_tongue: 

I interpret `.TS` as Table-Start and `.TE` as Table-End.
At the top is `tab(;);` which means: The columns will be separated by a semicolon in the markup. Notice how in the original it says `tab(@);`, that's why the columns are separated by `@` originally.
Then the `l l.` means both column have left-aligned text.
The `_` just adds the separating line below the header row
The rest is pretty self-explanatory.

I'm not sure what the 
```groff
T{
T}
```

in the original is for. I'm guessing that's useful if you want to use advanced markup in the cell contents themselves? But since you aren't doing that, it seems unnecessary.